app/presenters/summary/sections/base_section.rb

```diff
@@ -25,6 +25,15 @@ def name
         self.class.name.split('::').last.underscore.to_sym
       end
 
+      def title
```

Comment:
Move title from the partial. This is to allow summary card titles to become partner aware. Not also, the subject for summary cards now uses different content from the forms. (e.g. client and partner v your client and their partner, respectively)